googleMapsApiModule.py

```diff
@@ -53,6 +55,6 @@ def geocoding(search_location):
     print("Latitude: "+str(result[0]['geometry']['location']['lat'])+" "+"Longitude: "+str(result[0]['geometry']['location']['lng']))
     address = search_location.lower()
     address = address.replace(" ","+")
-    url = "https://www.google.com/maps/search/?api=1&query="
-    result_url = url+address
+    result_url = "{}&{}".format(BASE_URL['geocoding'], address.lower().replace(" ", "+"))
```

Comment:
Then, it'll be like this here:

```python3
result_url = "{}={}".format(BASE_URL['geocoding'], address.lower().replace(" ", "+"))
```

Reply:
@athityakumar we will address this in another issue, for now let us leave this as `result_url = url+address`

Reply:
Sure, works 👍